src/interfaces/token.spec.ts

```diff
@@ -789,28 +790,61 @@ describe("TokenInterface", () => {
       );
       expect(approveMock).toHaveBeenCalledTimes(1);
       expect(approveMock).toHaveBeenCalledWith(spenderAddress, amount, {});
-      expect(sendTransactionMock).toHaveBeenCalledTimes(1);
-      expect(sendTransactionMock).toHaveBeenCalledWith("approved");
     });
 
     it("should throw when approving native token", async () => {
       try {
-        await tokenInterface.approve(ownerAddress, ZeroAddress, spenderAddress, amount);
+        await tokenInterface.populateApprove(ownerAddress, ZeroAddress, spenderAddress, amount);
       } catch (error) {
         expect(error).toStrictEqual(new SdkError(`Native tokens cant be approved: ${ZeroAddress}`));
         expect(Contract).not.toHaveBeenCalled();
         expect(approveMock).not.toHaveBeenCalled();
-        expect(sendTransactionMock).not.toHaveBeenCalled();
```

Comment:
Is this suppose to be called?

Reply:
not on `populateApprove`. Just on `approve` now

Reply:
Same as the other comment, happy to not include, but still think it wouldn't hurt just to be sure.